src/tasks.ts

```diff
@@ -44,9 +43,8 @@ function createRunTask(t: Target, targetName: string) {
     },
     `Run ${targetDisplayName}`,
     "Meson",
-    new vscode.ProcessExecution(t.filename[0], {
-      cwd: workspaceState.get<string>("mesonbuild.sourceDir")!,
-      env: getEnvDict(),
```

Comment:
That means you can also delete getEnvDict function and _envDict related code in utils.ts.

Reply:
Whoops: makes sense, they have been removed :+1: